### PR TITLE
Update for 3.2.6.SP1-redhat-00001

### PR DESCRIPTION
**Pull request by llowinge**

### CODE REVIEW

**Comment by llowinge** [comment]
@ldrozdo Can you please review (mainly the docs). I'll tag Viliam later.

**Comment by vkasala** [comment]
Adding comment

**Comment by llowinge** [comment]
@vkasala Thanks. I'll merge after @ldrozdo approval.

**Comment by ldrozdo** [comment]
@llowinge LGTM, I just had some question about minor details but otherwise it can be merged.

File: README.md
Comment (ldrozdo) [reply README.md:110]: @llowinge do you see the review now?

File: README.md
Comment (llowinge) [reply README.md]: I've rewritten it.

File: README.md
Comment (llowinge) [reply README.md:110]: Yes, i've rewritten it.

**Review by ldrozdo — APPROVED** [review]

**Comment by ldrozdo** [comment]
@llowinge looks great, thanks

**Pull request merged**
